modules/ROOT/pages/redelivery-policy.adoc

```diff
@@ -0,0 +1,62 @@
+= Redelivery Policy
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, flow, redelivery, policy
+
+A redelivery policy is a filter that helps you conserve resources by limiting the number of times the Mule runtime engine (Mule) executes flow messages that cause errors. You can add a redelivery policy to some connectors that act as flow sources, such as the Salesforce, SAP, and VM connectors. To see whether a connector can use a redelivery policy, see the reference document for that connector.
```

Comment:
"flow messages that cause errors" :  
"flow messages" : unusual to use "flow" as an adjective this way
More importantly, the messages don't cause the errors. It's the delivery failure that would otherwise raise an error the first time it happens. 
Connectors are not sources. The source is a part of a connector. An HTTP listener is not an HTTP connector. It's part of the connector. HTTP connectors have other parts.

Reply:
"To see whether a connector can use a redelivery policy, see the reference document for that connector." <- We're going to get requests for a list of connectors that support redelivery policies.

Regardless, it would be good to provide some examples, links to the ref docs that point to the redelivery content for a couple of connectors.

Reply:
I do not want to list the connectors that have redelivery policies because they are constantly adding new connectors and new versions of connections. It is best to refer people to the connector docs for this.